storage/bucket.go

```diff
@@ -922,6 +928,7 @@ func (b *BucketAttrs) toRawBucket() *raw.Bucket {
 		Name:                  b.Name,
 		Location:              b.Location,
 		StorageClass:          b.StorageClass,
+		Updated:               b.Updated.Format(time.RFC3339),
```

Comment:
Since the field is read-only, it doesn't have to be included in `toRawBucket` or `toProtoBucket` I believe.

Reply:
Done